### DISCUSSION TITLE
Change CPU Target on Partial Compile

### DISCUSSION THREAD

**Discussion by djyt** [prb28/vscode-amiga-assembly#301] [Q&A] [answered]
Apologies if the topic header makes little sense.

My project is targeting the 68030 CPU. The Amiga Assembly framework can be changed in tasks to target the 68030 as follows on a full compile:

![image](https://github.com/prb28/vscode-amiga-assembly/assets/2414449/a6a5e00b-2f6a-46a4-bc9b-8b8e786d6d76)

However, when I save an individual file whilst developing, it still seems to target the plain 68000. How can I go about changing this? I've trawled through everywhere I can see and still can't find the source of this command:

![image](https://github.com/prb28/vscode-amiga-assembly/assets/2414449/8817c7d2-88fa-48c7-b53c-853f0dc1e1df)

**Comment by prb28** [comment, marked as answer]
Try this: https://github.com/prb28/vscode-amiga-assembly/wiki/Building#changing-the-build-options-for-syntax-checking

**Reply by djyt** [reply]
Magic, that worked thank you. I'd overlooked that part of the documentation.

**Reply by prb28** [reply]
It's not a trivial configuration. I have to make it more easy to set.
